docs/File-Formats.md

```diff
@@ -391,23 +391,24 @@ The expression metadata file should contain the following fields:
 #### Supported stable_id values for MRNA_EXPRESSION
 For historical reasons, cBioPortal expects the `stable_id` to be one of those listed in the following static set.
 
-datatype | stable_id | description
---- | --- | ---
-CONTINUOUS|mrna_U133|Affymetrix U133 Array
-Z-SCORE|mrna_U133_Zscores|Affymetrix U133 Array
-Z-SCORE|rna_seq_mrna_median_Zscores|RNA-seq data
-Z-SCORE|mrna_median_Zscores|mRNA data
-CONTINUOUS|rna_seq_mrna|RNA-seq data
-CONTINUOUS|rna_seq_v2_mrna|RNA-seq data
-Z-SCORE|rna_seq_v2_mrna_median_Zscores|RNA-seq data
-CONTINUOUS|mirna|MicroRNA data
-Z-SCORE|mirna_median_Zscores|MicroRNA data
-Z-SCORE|mrna_merged_median_Zscores|?
-CONTINUOUS|mrna|mRNA data
-DISCRETE|mrna_outliers|mRNA data of outliers
-Z-SCORE|mrna_zbynorm|?
-CONTINUOUS|rna_seq_mrna_capture|data from Roche mRNA Capture Kit
-Z-SCORE|rna_seq_mrna_capture_Zscores|data from Roche mRNA Capture Kit
+<table>
+<tr><td>datatype </td><td> stable_id </td><td> description</td></tr>
```

Comment:
I would suggest wrapping this row in a `<thead>`, and using `<th>`s for the cells instead of `<td>`. Or is there a reason not to? I do not have a local installation if Sphinx to test how it will look on Readthedocs, but the CommoMark spec explicitly mentions support for the keywords `thead` and `th`.